internal/api/v1/applications.go

```diff
@@ -11,10 +11,10 @@ import (
 	"github.com/epinio/epinio/helpers/kubernetes"
 	"github.com/epinio/epinio/helpers/kubernetes/tailer"
 	"github.com/epinio/epinio/helpers/tracelog"
-	"github.com/epinio/epinio/internal/api/v1/models"
 	"github.com/epinio/epinio/internal/application"
 	"github.com/epinio/epinio/internal/duration"
 	"github.com/epinio/epinio/internal/organizations"
+	"github.com/epinio/epinio/pkg/epinioapi/v1/models"
```

Comment:
I'm fine with just api, because I feel like the package path gives enough context. After reading 

* https://golang.org/doc/effective_go#package-names
* https://go.dev/blog/package-names

I felt like that was too generic. 

So, I thought what if the user used another API, what if we add another API to Epinio? How would we name the current API if there was another? For me it's (so far) always been the "Epinio API".

Reply:
Maybe it should be `v1/api`?

Reply:
If we are versioning the api then it should be `api/v1` (so that we can have `api/v2` in the future). If you reverse it, `v1` would refer to Epinio?

Reply:
I thought that we would force v2 to have `api` in it's path. 🤷  I see your point, makes it ambiguos what's being versioned